handlers/authorization/registration.py

```diff
@@ -12,55 +12,66 @@
 from datetime import datetime
 
 from translitua import translit
-from fastapi import APIRouter
-from fastapi.responses import JSONResponse
+from fastapi import APIRouter, status as http_status
+
+from schemas.jsend import JSENDOutSchema
+from components.exceptions import BackendException
 
 router = APIRouter()
 
 
-@router.post("/registration", response_model=RegistrationOut, tags=["Authorization"])
-async def registation(user: RegistrationIn):
+@router.post("/registration", response_model=JSENDOutSchema[RegistrationOut], tags=["Authorization"])
+async def registation(user: RegistrationIn):  # TODO syntax error
+
     RegistrationIn(
         token=user.token,
         email=user.email,
         password=user.password,
         password_re_check=user.password_re_check)
 
-    query = select(OneTimeToken).where(OneTimeToken.token == user.token)
+    query = select(OneTimeToken).where(OneTimeToken.c.token == user.token)
```

Comment:
Re-check this `.c` call, I gueess @YehorVoitenko removed it while refactor Models to declarative base.

Reply:
ok